frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -68,10 +69,10 @@ export const experimentLogic = kea<experimentLogicType>({
                 `api/projects/${teamLogic.values.currentTeamId}/insights`,
                 newInsight
             )
-            actions.setExperimentFunnel(createdInsight)
```

Comment:
this implied the experimentFunnel data we have here is stale: the filters don't match with the UI, nor do the results. So felt it's better to get rid of setting everything we don't need completely